### ISSUE TITLE
Fee reimbursement for trade 7218764

### ISSUE THREAD

**Issue by stonedbear** [bisq-network/support#1678]
SCREENSHOT HERE
![Screenshot from 2024-09-02 10-58-59](https://github.com/user-attachments/assets/dcf12c5d-3170-48b4-91f0-392c97ece554)

Maker:  66ce693f01706e137138357a6874f4c595b73e0150b8cf40aee5faed6fa01088
Taker: a431aba6461102d7e0f1ceafec32349f70a8d3cd8ef85c3bcfe0114e5aee2041
Deposit: 4c961ec9c1e8daf4c886cdc8546d0300b11e0fc3081d2f374002027cfb07f9af (did not show up in mempool)
BTC mining fees lost: Maker: 0.00001750 BTC / Taker: 0.00006990 BTC
Trade fees lost, please state if BSQ or BTC: Buyer: 0.0037 BTC / Seller: 0.0037 BTC

Bisq version: 1.9.17
BTC address for reimbursement: bc1qsm6xjy0h633wyyk70wwptqrvz9tgcrr75s7an6

Other notes:
Tried taking up a trade to buy BTC, multisig transaction was not showing up in the mempool for prolonged period of time. Did SPV resync - after which open trade claimed it is failed because of lack of multisig transaction.

**Comment by Mone167** [comment]
![Bildschirmfoto vom 2024-09-13 20-30-54](https://github.com/user-attachments/assets/88729db0-16c0-4928-854d-1d685c9955c4)
![Bildschirmfoto vom 2024-09-13 20-30-54](https://github.com/user-attachments/assets/a60c962b-1b11-42b7-8583-85408f54dd0a)
Version 1.9.17
Adresse für reimursement: bc1q2nwcr53x0wyyfv0gr43xqw6rs8nt6vpvr6jmse

**Comment by leo816** [comment]
Taker fee not found. No funds were lost in this trade.

let me know if I can help with anything else,

Kind regards,

**Comment by stonedbear** [comment]
Hi @leo816, thanks for looking into this. So it is a bit weirder that it seems at a first glance. (i did 100's of transactions on bisq and never had such an issue)

At first, when i was taking up a trade to buy bitcoin a transaction was created in bisq, but it never showed up in mempool.space.

I restatred bisq and did spv resync. After that the trade showed as failed but moreover i noticed entire traded amount missing, ie 0.0185 + buyer fees from my total balance in bisq wallet.

I did couple of more resync-s and the amount is still not showing up. So where did it go?

I had exactly 0.0427382 in my bisq wallet prior to the failed trade, and after it it showing 0.0204392. I have no transactions in my history it is as if it disappeared. The missing ammount equates to 0.0185 (the failed amount of to be purchased bitcoin and buyer fees)

Could it be that it went to some multisig limbo?
